webpack.prod.config.js

```diff
@@ -139,6 +140,7 @@ module.exports = (env = {}) => {
         },
       }),
       new webpack.ContextReplacementPlugin(/moment[/\\]locale$/, /en/),
+      new CopyWebpackPlugin(['assets/TermsOfService.html', 'assets/PrivacyPolicy.html', 'assets/RiskDisclaimerPolicy.html']),
```

Comment:
@mmvsha I think this is something we could easily forget. Can you move text content into a folder from root: `/content` and copy those into the build? Wdyt?